### PR TITLE
[sc-6455] Implement page to download activity

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: libs/common/src/assets/i18n/ca.json [hunk 1]
```diff
@@ -95,11 +95,13 @@
 	"accounts.no_users_found": "no existeix cap usuari amb aquest correu",
 	"accounts.users": "usuari/s",
 	"activity.actor": "Actor",
+	"activity.chat": "Preguntes i respostes generatives",
 	"activity.load-more": "TODO",
```
Comment (mpellerin42) [inline libs/common/src/assets/i18n/ca.json:99]: I see you didn't add it, but is this `load-more` used? if so you should replace the TODO by the proper translation and if not you can delete this entry

File: libs/common/src/assets/i18n/es.json [hunk 2]
```diff
@@ -95,11 +95,13 @@
 	"accounts.no_users_found": "No hemos encontrado a ningún usuario con este correo",
 	"accounts.users": "usuario/s",
 	"activity.actor": "Actor",
+	"activity.chat": "Preguntas y respuestas generativas",
 	"activity.load-more": "TODO",
```
Comment (mpellerin42) [inline libs/common/src/assets/i18n/es.json:99]: same for the spanish translation

**Review by mpellerin42 — APPROVED** [review]

File: libs/common/src/assets/i18n/ca.json (hunk 1 above)
Comment (operramon) [reply libs/common/src/assets/i18n/ca.json]: fixed

File: libs/common/src/assets/i18n/es.json (hunk 2 above)
Comment (operramon) [reply libs/common/src/assets/i18n/es.json]: fixed

**Pull request merged**